api/client/proto/authservice.proto

```diff
@@ -788,6 +788,13 @@ message DatabaseCertRequest {
     int64 TTL = 3 [ (gogoproto.jsontag) = "ttl", (gogoproto.casttype) = "Duration" ];
     // ServerNames are SANs to include in the certificate.
     repeated string ServerNames = 4 [ (gogoproto.jsontag) = "server_names" ];
+    // Sender is a name of service that sent the request.
+    enum Sender {
+        UNSPECIFIED = 0;
+        TCTL = 1;
+    }
+    // Requester is sender of the message.
+    Sender Requester = 5 [ (gogoproto.jsontag) = "requester" ];
```

Comment:
nit: I think I would prefer to use either `Sender` or `Requester`, but not two different names at the same time. just personal preference, you can keep if you like.